docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,193 +1,172 @@
 
-</TabItem>
-<TabItem label="OIDC">
+### Step 3/3
 
-Follow our [OIDC guides](./sso/oidc.mdx#identity-providers) to
-create an OIDC application.
+Assigning the Role to a User
 
-Copy the YAML below to a file called `oidc.yaml` and edit the information to
-include the details of your OIDC application.
+Now that you’ve created a role, assign it to a local user named Alice:
 
-```yaml
-(!examples/resources/oidc-connector.yaml!)
+```code
+$ tctl users add alice --roles=staging-access
 ```
 
-Create the `oidc` resource:
+Alice can now log in to staging servers, but she will be blocked from production environments.
+
+## Add local users with preset roles
+
+Teleport provides several preset roles:
+
+(!docs/pages/includes/preset-roles-table.mdx!)
+
+Invite the local user Alice as cluster `editor`:
 
 ```code
-$ tctl create okta.yaml
+$ tctl users add alice --roles=editor
 ```
 
-</TabItem>
-</Tabs>
+Invite the local user Alice as cluster `editor` and `reviewer`:
 
-### Teleport Community Edition
+```code
+$ tctl users add alice --roles=editor,reviewer
+```
 
-Save the file below as `github.yaml` and update the fields. You will need to
-set up a
-[GitHub OAuth 2.0 Connector](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/)
-app. Any member belonging to the GitHub organization `octocats` and on team
-`admin` will be able to assume the built-in role `access`.
+Once Alice signs up, she will be able to edit cluster configuration. You can list
+users and their roles using `tctl users ls`.
 
-```yaml
-kind: github
-version: v3
-metadata:
-  # connector name that will be used with `tsh --auth=github login`
-  name: github
-spec:
-  # client ID of GitHub OAuth app
-  client_id: client-id
-  # client secret of GitHub OAuth app
-  client_secret: client-secret
-  # This name will be shown on UI login screen
-  display: GitHub
-  # Change tele.example.com to your domain name
-  redirect_url: https://tele.example.com:443/v1/webapi/github/callback
-  # Map github teams to teleport roles
-  teams_to_roles:
-    - organization: octocats # GitHub organization name
-      team: admin            # GitHub team name within that organization
-      # map github admin team to Teleport's "access" role
-      roles: ["access"]
+
+```code
+$ tctl users ls
+
+# User                 Roles
+# -------------------- --------------
+# alice                editor, reviewer
+```
+
+You can update the user's roles using the `tctl users update` command:
+
+```code
+$ tctl users update alice --set-roles=editor,auditor
 ```
 
-Create the `github` resource:
+Once Alice logs back in, she will be able to view audit logs.
 
 ```code
-$ tctl create github.yaml
+$ tctl users update alice --set-roles=editor,reviewer,auditor
 ```
 
-## Step 3/3. Create a custom role
+Because Alice has two or more roles, permissions from those roles create a union. She will be able to act as a system administrator and auditor at the same time.
+
+## Create a custom role
```

Comment:
Same question as above re: the "Add local users with preset roles" section. I am not sure whether to treat this as a conceptual description or a how-to instruction. Is this an example of using label matching as the guide describes earlier?

Reply:
The idea was to provide basic steps to get started and to get new users familiar with these topics, such as labeling, and access.